collection_checklist.md

```diff
@@ -37,7 +37,7 @@ Note for reviewers: If you don't know how to check any of the points below, plea
 - [ ] `meta/runtime.yml` defines the minimal version of Ansible which the collection works with
 - [ ] has changelog, preferably with `changelogs/changelog.yaml`
 - [ ] collection repository should not contain any large objects (binaries) comparatively to the current Galaxy tarball size limit of 20 MB. For example, package installers for testing purposes shouldn't be added.
-- [ ] collection repository should not contain any unnecessary files like, for example, temporary files created by development tools. Temporary files MUST be added to `.gitignore`.
+- [ ] collection repository should not contain any unnecessary files like, for example, temporary files. Temporary files should be added to `.gitignore`.
```

Comment:
Why not must?

Reply:
There's no such corresponding must in the full set of rules, and to @felixfontein 's comment in https://github.com/ansible-community/community-topics/issues/138 , it's hard to use a MUST if we're being somewhat ambiguous about what counts as temporary files.

I was even debating taking out the `.gitignore` comment, or rewording to something like "consider adding to `.gitignore`" or whatever.

Happy to reconsider; if we want this as must, perhaps we should add it to the full criteria as well?

Reply:
I'm not sure how "must" actually makes sense here. Would that mean that every possible kind of temporary file must be mentioned in `.gitignore`? I'm wondering how one would actually do that :) In that case "should" is better than "must", but "consider adding to `.gitignore`" is even better than "should".

Reply:
> it's hard to use a MUST if we're being somewhat ambiguous about what counts as temporary files.

That's a good point. I think `should` makes sense here then.